lib/Elastica/Multi/Search.php

```diff
@@ -27,18 +32,20 @@ class Search
     protected $_options = array();
 
     /**
-     * @var \Elastica\Client
+     * @var array|\Elastica\Search[]
      */
-    protected $_client;
+    protected $_searches = array();
 
     /**
      * Constructs search object.
      *
      * @param \Elastica\Client $client Client object
+     * @param MultiBuilderInterface $builder
      */
-    public function __construct(Client $client)
+    public function __construct(Client $client, MultiBuilderInterface $builder = null)
     {
-        $this->setClient($client);
+        $this->_builder = $builder ?: new MultiBuilder();
```

Comment:
I think that in the case of a Multi\Search, we're not going to encounter many people overriding this builder. There's just not much need to customise what it does, but providing that capability means noone has to extend core classes anymore.

The MultiBuilder is implemented that each Search that it runs can have its own builder, which is why I've left the `MultiBuilderInterface` and `MultiBuilder` in the Multi namespace.

I cant really remove the MultiBuilder concept - it has a difference purpose to the ResultSet Builder. There may be a better name for what it is doing?